src/main/java/seedu/Parser/ParseEdit.java

```diff
@@ -0,0 +1,37 @@
+package seedu.Parser;
+
+import seedu.commands.Command;
+import seedu.commands.EditCommand;
+import seedu.commands.InvalidCommand;
+
+public class ParseEdit {
+    public static Command editItem(String line) {
+        // Format: edit, index, d/date, a/amount, s/description
+        int space = line.indexOf(" ");
+        String[] allDetails = SplitCommand.splitCommand(space, line);
+        String details = allDetails[1];
+
+        int posDSlash = details.indexOf('/');
+        String[] furtherDetails = SplitCommand.splitCommand(posDSlash, details);
+        String indexVal = furtherDetails[0];
```

Comment:
Moving forward can use constants to denote this number to avoid magic literals

Reply:
noted, will change it in my next update